vignettes/archive.Rmd

```diff
@@ -377,18 +375,16 @@ fc <- bind_rows(k_week_ahead(x, ahead = 7, as_of = TRUE),
                 k_week_ahead(x, ahead = 28, as_of = FALSE))
 
 # Plot them, on top of latest COVID-19 case rates 
-p <- ggplot(fc, aes(x = target_date, group = time_value, fill = as_of)) +
+ggplot(fc, aes(x = target_date, group = time_value, fill = as_of)) +
   geom_ribbon(aes(ymin = fc_lower, ymax = fc_upper), alpha = 0.4) +
   geom_line(aes(y = fc_point)) + geom_point(aes(y = fc_point), size = 0.5) + 
   geom_vline(aes(xintercept = time_value), linetype = 2, alpha = 0.5) +
   facet_grid(vars(geo_value), vars(as_of), scales = "free") +
   scale_x_date(minor_breaks = "month", date_labels = "%b %y") +
   labs(x = "Date", y = "Reported COVID-19 case rates") + 
-  theme(legend.position = "none")
-  
-gginnards::append_layers(
-  p, geom_line(data = x_latest, aes(x = time_value, y = case_rate), 
-               inherit.aes = FALSE, color = "gray50"), pos = "bottom")
+  theme(legend.position = "none") +
+  geom_line(data = x_latest, aes(x = time_value, y = case_rate), 
```

Comment:
Given that the `gginnards` code says "bottom", shouldn't this line come *first*? Right after the `ggplot()` call?